doc/source/deployment/setup-deployer.rst

```diff
@@ -68,22 +68,37 @@ Deployer. Set up your workspace with the following steps:
   export SOCOK8S_WORKSPACE_BASEDIR=~
 
 
-Cloning the repository
-----------------------
+Installing the SUSE Containerized OpenStack software
+----------------------------------------------------
 
-To get started, clone the `socok8s GitHub repository <https://github.com/SUSE-Cloud/socok8s>`_.
-This repository uses submodules, so you need to get all the code to make sure
-the playbooks work.
+There are several ways to install the SUSE Containerized OpenStack software.
 
-::
+1. Install from the openSUSE repository including required dependencies:
 
-   git clone --recursive https://github.com/SUSE-Cloud/socok8s.git
+   a. sudo zypper addrepo \\
+      https://download.opensuse.org/repositories/Cloud:/socok8s/openSUSE_Leap_15.0/ socok8s
+   b. sudo zypper install socok8s (installs to /usr/share/socok8s)
 
-Alternatively, one can fetch/update the tree of the submodules by running:
+2. Install with an ISO image including required dependencies:
```

Comment:
One more nit: could we please move ISO as the first preferred approach we agreed with the PM and architect.

Reply:
Right, the iso should be available elsewhere for download.
But I would think documentation could be adapted once we know the right location